src/uct/rocm/copy/rocm_copy_md.c

```diff
@@ -22,17 +22,18 @@
 #include <uct/api/v2/uct_v2.h>
 #include <hsa_ext_amd.h>
 
-static ucs_config_field_t uct_rocm_copy_md_config_table[] = {
-    {"", "", NULL,
-     ucs_offsetof(uct_rocm_copy_md_config_t, super),
-     UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
+static ucs_config_field_t uct_rocm_copy_md_config_table[] =
+        {{"", "", NULL, ucs_offsetof(uct_rocm_copy_md_config_t, super),
```

Comment:
pls use the previous code alignment

Reply:
ok, I will revert the last change that introduced the format changes to this block, but just for clarification, the only reason I made this change was because the code formatting checker otherwise always reported 'passed with issues',